Configuration/EventContent/python/EventContent_cff.py

```diff
@@ -95,7 +95,14 @@
 #
 #
 from HLTrigger.Configuration.HLTrigger_EventContent_cff import *
-#
+from HLTrigger.Configuration.HLTScouting_EventContent_cff import HLTScoutingAll
+# Extend HLT dataformats to the previous scouting objects, to keep them when running on old data/MC
+HLTriggerMINIAOD.outputCommands.extend(HLTScoutingAll.outputCommands)
+HLTriggerAOD.outputCommands.extend(HLTScoutingAll.outputCommands)
+HLTriggerRECO.outputCommands.extend(HLTScoutingAll.outputCommands)
+HLTriggerRAW.outputCommands.extend(HLTScoutingAll.outputCommands)
+HLTDebugRAW.outputCommands.extend(HLTScoutingAll.outputCommands)
+HLTDebugFEVT.outputCommands.extend(HLTScoutingAll.outputCommands)
```

Comment:
Hmm, why do you add this to more than just MINIAOD and AOD? Eg, RAW as in data taking is always current products only, no point in adding previous years.

Reply:
I don't have a strong opinion and I don't have a clear use case for this. The approach was to extend the scouting collections to the past collections everywhere for simplicity. We might catch some strange corner cases. Typically FEVT is more inclusive than RECO, and RECO is more inclusive than AOD.
What about keep it for `HLTriggerRECO`, `HLTDebugFEVT`, and drop it for `HLTriggerRAW` and `HLTDebugRAW` ?

Reply:
Yeah in general, but not for scouting products: they are added in for AOD/MINIAOD only, but not included in RECO/FEVT, no?